### PR TITLE
Add sepolicy for sensor mediation

### PR DESCRIPTION
**Pull request by JaikrishnaNemallapudi**
Tracked-On: OAM-91265
Signed-off-by: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>

### CODE REVIEW

File: sensors/mediation/file_contexts [hunk 1]
```diff
@@ -0,0 +1 @@
+/(vendor|system/vendor)/bin/hw/android\.hardware\.sensors\.intel@1\.0-service u:object_r:hal_sensors_default_exec:s0
```
Comment (JeevakaPrabu) [inline sensors/mediation/file_contexts]: If google provided service is not going to be modified, please use google provided service

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]
Detail the changes in commit message

File: sensors/mediation/file_contexts (hunk 1 above)
Comment (JaikrishnaNemallapudi) [reply sensors/mediation/file_contexts]: Updated sensor service name to google default

**Review by ceiba1985 — APPROVED** [review]

File: sensors/mediation/file_contexts [hunk 2]
```diff
@@ -0,0 +1 @@
+/(vendor|system/vendor)/bin/hw/android\.hardware\.sensors@1\.0-service u:object_r:hal_sensors_default_exec:s0
```
Comment (JeevakaPrabu) [inline sensors/mediation/file_contexts]: this is not needed as Google should already have this sepolicy

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]

**Review by JeevakaPrabu — APPROVED** [review]

**Review by ceiba1985 — APPROVED** [review]

**Review by kwang13 — APPROVED** [review]

**Pull request merged**